flask_resources/serializers/json.py

```diff
@@ -44,7 +44,7 @@ def default(self, obj):
 class JSONSerializer(BaseSerializer):
     """JSON serializer implementation."""
 
-    def __init__(self, encoder=None, options=None):
+    def __init__(self, encoder=None, options=None, **kwargs):
```

Comment:
This change was neeed due to adding positional arguments from `**kwargs` before. Might not be needed if we change the way the schema is initialised.